frontend/packages/kubevirt-plugin/src/components/modals/disk-modal/disk-modal.tsx

```diff
@@ -172,7 +173,13 @@ export const DiskModal = withHandlePromise((props: DiskModalProps) => {
     type,
   });
 
-  const resultDataVolumeName = prefixedID(vmName, name);
+  // We can generate a random name every time, because this modal should not operate on disks with live datavolumes
+  const resultDataVolumeName = resolveDataVolumeName({
+    diskName: name,
+    vmLikeEntityName: vmName,
+    isTemplate,
+    isPlainDataVolume: isInWizard && source.isPlainDataVolume(isTemplate),
```

Comment:
```suggestion
    isPlainDataVolume: source.isPlainDataVolume(isInWizard && isTemplate),
```

This would also conform to the naming in isPlainDataVolume which is called isCreateTemplate

Reply:
can we also extract a variable out of that so we can use it in the other place?